### PR TITLE
Add support for grunt-php, to use as standalone development server

### PR DESCRIPTION
**Pull request by basst85**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -4,30 +4,42 @@ module.exports = function(grunt) {
...
-            dist: {
-                files: {
-                    "web/styles/app.css": "resources/Sass/all.scss"
-                }
-            }
-        }
+		php: {
+			dist: {
+				options: {
+					hostname: 'YOUR_IP_HERE',
```
Comment (KeizerDev) [inline Gruntfile.js]: Please change to `127.0.0.1` or use an environment file where you can define it

Comment (basst85) [reply Gruntfile.js]: Good point!
But 127.0.0.1 does not work when you work on a external server/VPS 😉 (ok yes, it can work with VPN..)

Comment (KeizerDev) [reply Gruntfile.js]: Then add vagrant :stuck_out_tongue:

Comment (DennisSnijder) [reply Gruntfile.js]: Nice PR, however I do agree with @KeizerDev about this one, it would be nice to use `127.0.0.1` as a default IP.

**Comment by DennisSnijder** [comment]
Works like a charm @basst85 ! 😄 
and thanks @KeizerDev for your feedback 👍

**Pull request merged**